jfr-streaming/src/main/java/io/opentelemetry/contrib/jfr/metrics/network/PerThreadNetworkReadHandler.java

```diff
@@ -0,0 +1,55 @@
+package io.opentelemetry.contrib.jfr.metrics.network;
+
+import io.opentelemetry.api.common.AttributeKey;
+import io.opentelemetry.api.common.Attributes;
+import io.opentelemetry.api.metrics.BoundDoubleHistogram;
+import io.opentelemetry.api.metrics.Meter;
+import io.opentelemetry.contrib.jfr.Constants;
+import io.opentelemetry.contrib.jfr.metrics.RecordedEventHandler;
+import jdk.jfr.consumer.RecordedEvent;
+
+public class PerThreadNetworkReadHandler implements RecordedEventHandler {
+  private static final String DESCRIPTION_BYTES = "Bytes Read";
+  private static final String DESCRIPTION_DURATION = "Read Duration";
+  private final String threadName;
+
+  public static final String JFR_SOCKET_READ_DURATION = "jfr.SocketRead.duration";
+  public static final String JFR_SOCKET_READ_BYTES_READ = "jfr.SocketRead.bytesRead";
```

Comment:
Should stick to [system conventions](https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/metrics/semantic_conventions/system-metrics.md#systemnetwork---network-metrics) a bit closer.


Suggestion:   `runtime.jvm.network.io` with attribute denoting "read"

For duration, `runtime.jvm.network.duration`